index.html

```diff
@@ -23,80 +24,79 @@
-                <img class="full-image" src="https://placeimg.com/1280/720/nature">
-            </a>
-
-            <a href="#">
-                <img class="thumbnail" src="https://placeimg.com/200/200/nature">
-                <img class="full-image" src="https://placeimg.com/1280/720/nature">
-            </a>
-
-            <a href="#">
-                <img class="thumbnail" src="https://placeimg.com/200/200/nature">
-                <img class="full-image" src="https://placeimg.com/1280/720/nature">
-            </a>
-
-            <a href="#">
-                <img class="thumbnail" src="https://placeimg.com/200/200/nature">
-                <img class="full-image" src="https://placeimg.com/1280/720/nature">
-            </a>
-
-            <a href="#">
-                <img class="thumbnail" src="https://placeimg.com/200/200/nature">
-                <img class="full-image" src="https://placeimg.com/1280/720/nature">
-            </a>
-        </div>
+        <button class="carousel__button carousel__button--right">
+        </button>
     </section>
 
+            <aside>
+                <div class="thumbnails">
+
+                    <div class="aside aside1">
+                        <a href="#">
+                            <img class="thumbnail" src="https://placeimg.com/200/200/nature">
+                        </a>
+                        <h3>Something something</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Voluptatum accusamus, rerum architecto repellendus laboriosam magni!</p>
+                    </div>
+                    
+                    <div class="aside aside2">
+                        <a href="#">
+                            <img class="thumbnail" src="https://placeimg.com/200/200/nature">
+                        </a>
+                        <h3>Something something</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Voluptatum accusamus, rerum architecto repellendus laboriosam magni!</p>
+                    </div>
+
+                    <div class="aside aside3">
+                        <a href="#">
+                            <img class="thumbnail" src="https://placeimg.com/200/200/nature">
+                        </a>
+                        <h3>Something something</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Voluptatum accusamus, rerum architecto repellendus laboriosam magni!</p>
+                    </div>
+                    
+                </div>
+            </aside>
+
     <footer>
         <div class="footer-col">
             <h4>Guide</h4>
-            <ul>
-                <li><a href="#">Values</a></li>
-                <li><a href="#">Mission</a></li>
-                <li><a href="#">Strategy</a></li>
-                <li><a href="#">Todo better world</a></li>
-            </ul>
+            <div class="footer-links">
+                <a href="#">Values</a>
```

Comment:
styling could be added to a tag in the footer. It's hard to read the blue text in black background.